client/jest.config.js

```diff
@@ -15,6 +15,7 @@ const config = {
   moduleNameMapper: {
     ...pathsToModuleNameMapper(compilerOptions.paths, { prefix: '<rootDir>/' }),
     ...jestModuleMapper,
+    "\\.(css|less)$": "<rootDir>/../node_modules/@patternfly/react-styles/__mocks__/styleMock.js"
```

Comment:
I thought `jestModuleMapper` brought in fake file support - will it work if you augment `/utils/tooling/aliasHelper.js` to include `less` instead of bringing in `styleMock`?